clarifai/client/app.py

```diff
@@ -236,30 +242,93 @@ def create_model(self, model_id: str, **kwargs) -> Model:
 
     return Model(model_id=model_id, **kwargs)
 
-  def create_workflow(self, workflow_id: str, **kwargs) -> Workflow:
+  def create_workflow(self, config_filepath: str, generate_new_id: bool = False) -> Workflow:
```

Comment:
can this arg be `generate_random_id` instead of `generate_new_id`? WDYT?

Reply:
it's definitely closer to the actual implementation, so I'm for it as well.

Reply:
Even thought it just generates random id. I think `generate_new_id` is more intuitive